### ISSUE TITLE
Missing Stormwind Spirit Healer

### ISSUE THREAD

**Issue by LordUsagi** [OregonCore/OregonCore#1403]
**Current behavior**: 
The "graveyard" you spawn at when you die in Stormwind as Alliance shows a Spirit Healer icon on the minimap but the Spirit Healer is missing.

**Expected behavior**: 
The Spirit Healer should be present

**Steps to reproduce the problem**:
Die in Stormwind. Release Spirit.

**Comment by Meltie2013** [comment]
@LordUsagi Are you talking about the one south of Stormwind, Northwest of Goldshire?

**Comment by LordUsagi** [comment]
Kinda. I'll post screenshots. I'm definitely aware there's one closer to
goldshire but where I'm ghosting too is more near the stormwind duel zone

**Comment by Meltie2013** [comment]
Ah, that one. I checked other projects and its the same thing. I'll pull a sniff here to see what's going on.

**Comment by LordUsagi** [comment]
Yeah, I'm really not sure if one is supposed to be there but find it odd that that is where I spawn and that my minimap mentions spirit healer!

**Comment by Meltie2013** [comment]
After looking through the WorldSafeLoc.dbc, it appears to be one there. No data is available at this time in the current sniffs I have. 

From WorldSafeLoc.dbc

```
Graveyard Id [107]
GhostZone    [1519]
Comment      [Elwynn Forest, Stormwind]
```

**Comment by Meltie2013** [comment]
@LordUsagi After some researching most data banks are not complete, but an old friend of mine, does remember a Spirit Healer NPC in that same location back in Wrath of the Lich King. Sniff data for that location, is very small to non-existent. I'll do a small sniff on retail to get the correct coords.
